src/Serilog.Sinks.RavenDB/Sinks/RavenDB/RavenDBSink.cs

```diff
@@ -47,12 +52,26 @@ public class RavenDBSink : PeriodicBatchingSink
         /// <param name="batchPostingLimit">The maximum number of events to post in a single batch.</param>
         /// <param name="period">The time to wait between checking for event batches.</param>
         /// <param name="formatProvider">Supplies culture-specific formatting information, or null.</param>
-        public RavenDBSink(IDocumentStore documentStore, int batchPostingLimit, TimeSpan period, IFormatProvider formatProvider)
+        /// <param name="defaultDatabase">Optional name of default database</param>
+        /// <param name="expirationTimeSpan">Optional time before a logged message will be expired assuming the expiration bundle is installed. Zero (00:00:00) means no expiration. If this is not provided but errorExpirationTimeSpan is, errorExpirationTimeSpan will be used for non-errors too.</param>
```

Comment:
Rather than `Zero`, `Timeout.InfiniteTimeSpan` is often used to communicate a "never" timeout. May be a better choice than zero here?